ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -8,6 +8,8 @@ import CoreGraphics
 
 class InterceptingSpec: QuickSpec {
 	override func spec() {
+		ForwardInvocationTestObject.swizzleForTesting()
```

Comment:
I'd put this in `beforeAll`. If these tests aren't gonna run there's no point in calling this function.